ui/store/actions.ts

```diff
@@ -31,6 +37,7 @@ import {
   hasTransactionPendingApprovals,
   ///: BEGIN:ONLY_INCLUDE_IN(snaps)
   getNotifications,
+  getPermissionSubjects,
   ///: END:ONLY_INCLUDE_IN
 } from '../selectors';
```

Comment:
This is also fenced in `snaps`, not sure if it's correct for this particular case